app/views/layouts/application.html.haml

```diff
@@ -5,6 +5,8 @@
 /[if IE 9 ]    <html lang="en" class="no-js ie9">
 <!--[if (gt IE 9)|!(IE)]><!--> <html lang="en" class="no-js"> <!--<![endif]-->
 %head
+  - if staging_env?
+    %meta{ name: "robots", content: "noindex" }/
```

Comment:
is the `/` needed? It's not in the other meta in this file

Reply:
@Br3nda I don't think it's needed (the browser will automatically add it if it's missing) but I think it's best practice to put a trailing slash if it's a self-closing tag, for semantic reasons